explorer/src/providers/stats/solanaDashboardInfo.tsx

```diff
@@ -0,0 +1,96 @@
+import { PerfSample } from "@solana/web3.js";
+import { ClusterStatsStatus } from "./solanaClusterStats";
+
+export type DashboardInfo = {
+  status: ClusterStatsStatus;
+  avgBlockTime_1h: number;
+  avgBlockTime_1min: number;
+  epochInfo: {
+    absoluteSlot: number;
+    blockHeight: number;
+    epoch: number;
+    slotIndex: number;
+    slotsInEpoch: number;
+  };
+};
+
+export enum DashboardInfoActionType {
+  SetPerfSamples,
+  SetEpochInfo,
+  SetError,
+  Reset,
+}
+
+export type DashboardInfoActionSetPerfSamples = {
+  type: DashboardInfoActionType.SetPerfSamples;
+  data: PerfSample[];
+};
+
+export type DashboardInfoActionSetEpochInfo = {
+  type: DashboardInfoActionType.SetEpochInfo;
+  data: any;
```

Comment:
Any reason why this isn't typed?

Reply:
No. I think it may have been untyped when I moved it over. Maybe not; I'll fix!